libdnf/conf/ConfigRepo.cpp

```diff
@@ -174,6 +177,14 @@ ConfigRepo::Impl::Impl(Config & owner, ConfigMain & mainConfig)
     owner.optBinds().add("enabled_metadata", enabled_metadata);
     owner.optBinds().add("user_agent", user_agent);
     owner.optBinds().add("countme", countme);
+    owner.optBinds().add("failovermethod", failovermethod,
+        [&](Option::Priority priority, const std::string & value){
+            if (value != "priority") {
+                throw Option::InvalidValue(tfm::format(_("only the value 'priority' is supported.")));
```

Comment:
Do we really need to call `tfm::format` and `#include "tinyformat/tinyformat.hpp"`?

`throw Option::InvalidValue(_("only the value 'priority' is supported."));`

Reply:
Right, thanks. That's the remnant of the previous version of the patch that used to print more information. I will remove it.